lib/services/presets.go

```diff
@@ -151,10 +151,29 @@ func NewPresetAuditorRole() types.Role {
 				Rules: []types.Rule{
 					types.NewRule(types.KindSession, RO()),
 					types.NewRule(types.KindEvent, RO()),
+					types.NewRule(types.KindSessionTracker, RO()),
 				},
 			},
 		},
 	}
 	role.SetLogins(types.Allow, []string{"no-login-" + uuid.New().String()})
 	return role
 }
+
+func UpdateAuditorRoleRFD82(role types.Role) types.Role {
+	found := false
+	for _, rule := range role.GetRules(types.Allow) {
+		if rule.Resources[0] == types.KindSessionTracker {
+			found = true
+			break
```

Comment:
You could get rid of `found` entirely and just `return role` here since there is no additional work to be done.